Client/src/app.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import products from './products.js'
+import '../dist/style.css';
+import $ from 'jquery';
+import ProductView from '/components/ProductView.jsx'
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+    test: "test",
+    currentProduct:1,
+    productInfo= []
+    }
+  //bind event handler
+    this.getProductInfo=this.getProductInfo.bind(this);
+  }
+  //event handlers
+getProductInfo() {
```

Comment:
No error handling here which looks incomplete.  You may have gotten to this by now but if not lets make sure we don't forget to tackle it.